### PR TITLE
[Samples][40.timex-resolution] Migrate README document

### PR DESCRIPTION
**Pull request by FedericoBernal**
Fixes #

## Specific Changes

  - 
  -
  -

### CODE REVIEW

File: samples/40.timex-resolution/README.md [hunk 1]
```diff
@@ -0,0 +1,357 @@
+# Timex Resolution
+
+This bot has been created using [Bot Framework](https://dev.botframework.com), is shows how to use TIMEX expressions.
```
Comment (matiasroldan6) [inline samples/40.timex-resolution/README.md:3]: _it_ shows?

Comment (FedericoBernal) [reply samples/40.timex-resolution/README.md:3]: good catch!

File: samples/40.timex-resolution/README.md [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+
+Can be resolved with the constraint of T14 to get to the specific date time of:
+
+```
+2020-07-01T14
+```
+
+## TIMEX resolution using the TimexExpressions library
+
+The approach to dealing with TIMEX expressions outlines in above is implemented in the [TimexExpressions](https://github.com/microsoft/Recognizers-Text/tree/master/Java/libraries/recognizers-text-datatypes-timex-expression) library current available for .NET Java and JavaScript. And examples of its usage are included in this sample.
```
Comment (Batta32) [inline samples/40.timex-resolution/README.md:322]:
```suggestion
The approach to dealing with TIMEX expressions outlines in above is implemented in the [TimexExpressions](https://github.com/microsoft/Recognizers-Text/tree/master/Java/libraries/recognizers-text-datatypes-timex-expression) library current available for .NET, Java and JavaScript. And examples of its usage are included in this sample.
```

File: samples/40.timex-resolution/README.md [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+- From the root of this project folder:
+    - Build the sample using `mvn package`
+    - Run it by using `java -jar .\target\bot-timex-resolution-sample.jar`
+
+## Experimenting with Recognizers
+
+TIMEX has been introduced into the bot authoring platform by the [Text Recognizers](https://github.com/Microsoft/Recognizers-Text) package. One of the best ways to understand the TIMEX behavior is to experiment directly with the Rcognizers. You can do this by install the appropriate Recognizer package, instantiating the date time recognizer and calling it with your test string. In Node you can use the following 3 lines of code, but the behavior is identical in C#, Python and Java.
+
+```
+  Recognizer recognizer = require("@microsoft/recognizers-text-date-time")
```
Comment (Batta32) [inline samples/40.timex-resolution/README.md:40]: Is this ok? I don't think so

File: samples/40.timex-resolution/README.md [hunk 4]
```diff
@@ -0,0 +1,359 @@
...
+  String result = recognizer.recognize("next Wednesday 4pm", Culture.English);
```
Comment (Batta32) [inline samples/40.timex-resolution/README.md:41]:
```suggestion
  String result = recognizer.recognize("next Wednesday 4pm", Culture.English);
```

In C# this is a `recognizeDateTime`, do we have this?

File: samples/40.timex-resolution/README.md [hunk 5]
```diff
@@ -0,0 +1,359 @@
...
+
+## To try this sample
+
+- From the root of this project folder:
+    - Build the sample using `mvn package`
+    - Run it by using `java -jar .\target\bot-timex-resolution-sample.jar`
+
+## Experimenting with Recognizers
+
+TIMEX has been introduced into the bot authoring platform by the [Text Recognizers](https://github.com/Microsoft/Recognizers-Text) package. One of the best ways to understand the TIMEX behavior is to experiment directly with the Rcognizers. You can do this by install the appropriate Recognizer package, instantiating the date time recognizer and calling it with your test string. In Node you can use the following 3 lines of code, but the behavior is identical in C#, Python and Java.
```
Comment (Batta32) [inline samples/40.timex-resolution/README.md:37]: Here it says `3 lines of code`, we should change this as soon as we finish

File: samples/40.timex-resolution/README.md [hunk 6]
```diff
@@ -0,0 +1,359 @@
...
+
+```
+  Recognizer recognizer = require("@microsoft/recognizers-text-date-time")
+  String result = recognizer.recognize("next Wednesday 4pm", Culture.English);
+  JSONObject obj = (result, null, 2);
+  String JSON = obj.toString();
+  System.out.println(JSON);
```
Comment (Batta32) [inline samples/40.timex-resolution/README.md:44]: Not sure about this implementation, we have never used `JSONObject`

**Review by Batta32 — CHANGES_REQUESTED** [review]

**Review by Batta32 — APPROVED** [review]

**Pull request merged**